test/fixtures/use-strict.upgraded.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+var director = require('./resolved/director');
```

Comment:
I'd like to get another line-break here, but I end up getting 2 more when I mess with it.